docs/targets/awscomprehend.md

```diff
@@ -0,0 +1,33 @@
+# Event Target for AWS Comprehend
+
+This event target receives [CloudEvents][ce] over HTTP and sends them to [AWS Comprehend](https://aws.amazon.com/comprehend/)
+
+## Prerequisite(s)
+
+- AWS API key and secret.
+
+## Deploying an Instance of the Target
+
+Open the Bridge creation screen and add a target of type `AWS Comprehend`.
+
+In the Target creation form, give a name to the event Target and add the following information:
+
+- **AWS Secret**: Reference a [TriggerMesh secret](../guides/secrets.md) containing an AWS API key and Secret.
+- **Region**: Denotes the region to run the AWS Comprehend services from.
+- **Language**: Denotes the language to expect in the events.
+
+After clicking the `Save` button, you will be taken back to the Bridge editor. Proceed to adding the remaining
+components to the Bridge, then submit it.
+
+A ready status on the main _Bridges_ page indicates that the AWS Comprehend target is ready to accept events.
+
+For more information about using AWS Comprehend, please refer to the [documentation][https://docs.aws.amazon.com/comprehend/].
+
+## Event Types
+### Arbitrary
+This target will consume arbitrary events and analyzes each of the key values sentiment. It then combines the scores and 
+returns the analysis in a response event of type `io.triggermesh.targets.aws.comprehend.result`
+
+
+[ce]: https://cloudevents.io/
+[ce-jsonformat]: https://github.com/cloudevents/spec/blob/v1.0/json-format.md
```

Comment:
if that footnote thing does work... the ce-jsonformat isn't used?

Reply:
no that is copy pasta good catch

Reply:
updated :)